inference-engine/src/gna_plugin/backend/make_pwl.cpp

```diff
@@ -527,7 +593,7 @@ void make_gna_pwl(const DnnActivation  fun,
                         0 });
 
                     gnalog() << (x_upper & XBASEMASK) / in_scale
-                        << " " << gna_pwl[2].yBase / out_scale
+                        << " " << gna_pwl[n_segments].yBase / out_scale
```

Comment:
`gna_pwl[n_segments]` looks risky
shouldnt it be `gna_pwl[n_segments - 1]`

Reply:
It's needed that way for inserting _at_ and _before_ the last position.